container.nix

```diff
@@ -87,7 +95,7 @@ let
     };
 
     environments = quadletUtils.mkOption {
-      type = types.attrs;
+      type = types.attrsOf types.str;
```

Comment:
This might break existing configuration for some users, but IMO it's worth enforcing the `str`, since `toString` is unreliable. For example `toString` with boolean produces `""` for `false` and `"1"` for `true`. 🤯 
![image](https://github.com/user-attachments/assets/07b08ff9-fcac-4433-a586-7ae23ecc7b2a)

Reply:
Good catch!

TIL 🤯.